gdal/build.sh

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
+./configure --with-python --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
+--with-static-proj4=$PREFIX \
+--with-hdf5=$PREFIX \
+--with-hdf4=$PREFIX \
+--with-netcdf=$PREFIX \
+--with-xerces=$PREFIX \
+--with-sqlite=$PREFIX \
```

Comment:
IIRC (I am not at my work computer where I was doing some experiments), --with-sqlite isn't an actual switch for configure.